### DISCUSSION TITLE
Local avatars seem to be missing after failed data move attempt

### DISCUSSION THREAD

**Discussion by samip5** [mastodon/mastodon#22733] [Q&A]
What is the local path the local avatars are stored?

I don't seem to have `live/public/system/avatars` but should I even or where are they supposed to be served from?

**Comment by nightpool** [comment]
is that folder there on your old server? how did you move the data?

it depends on how your old server was set up, whether it was using the filesystem or object storage.

**Reply by samip5** [reply]
No object storage were ever used, and that whole folder was moved to another location aka live/public/system but no local avatars are loading even after reverting that change so were the local avatars there too or are they stored somewhere else on disk?

There is no old server, they are the same machine but just tweaked with mounts.
